components/form-builder/app/responses/DownloadTable.tsx

```diff
@@ -250,7 +257,7 @@ export const DownloadTable = ({
                         }
                       }}
                       setDownloadError={setDownloadError}
-                      setShowConfirmNewDialog={setShowConfirmNewDialog}
+                      ariaLabelledBy={submission.name}
```

Comment:
Currently this is announced as Label + Accessible name or e.g. 16-11-0bd7, button

I wonder if adding the word download in there would make it more obvious? Maybe something like:
` ariaLabelledBy={submission.name + " download}`
which a screen reader would announce as: 16-11-0bd7 download, button

Though the table relationship exists so a user could/should be able to figure it out. Both work

Reply:
Wops, that should have been an array of elements